### PR TITLE
Feature/grv-1955-post-with-id

### PR DESCRIPTION
**Pull request by gfellerleonard**
The problem was that it did an UPDATE request when the user did a POST request with an ID in the payload, what is actually not right. This should solve the problem and returns an error message by validating the request.

### CODE REVIEW

**Comment by narcoticfresh** [comment]
(i want to hold this back until we have a stable release for bap)..

**Comment by hairmare** [comment]
I'm :+1: on this change and don't feel we should be holding it back for any reason whatsoever.

This PR still needs to add some tests to show that it really is working...

File: src/Graviton/CoreBundle/Tests/Controller/AppControllerTest.php [hunk 1]
```diff
@@ -245,6 +245,30 @@ public function testPostMalformedApp()
...
+        $helloApp->name = "tubel";
+
+        $client = static::createRestClient();
+        $client->post('/person/customer', $helloApp);
+        $response = $client->getResponse();
+        $results = $client->getResults();
+
+        $this->assertEquals(
+            'Can not be given on a POST request. Do a PUT request instead to update an existing record.',
+            json_decode($response->getContent())[0]->message
```
Comment (hairmare) [inline src/Graviton/CoreBundle/Tests/Controller/AppControllerTest.php]: You can use ``$results[0]->message`` that already does the decoding.

**Comment by RamonB** [comment]
new https://github.com/libgraviton/graviton/pull/188

**Pull request closed without merging**